lib/Auth/Process/serverconfig.php

```diff
@@ -32,6 +32,7 @@ public function __construct( array $config, $reserved ) {
 		$this->serverconfig['doTriggerChallenge'] = $cfg->getBoolean('doTriggerChallenge', false);
 		$this->serverconfig['tryFirstAuthentication'] = $cfg->getBoolean('tryFirstAuthentication', false);
 		$this->serverconfig['tryFirstAuthPass'] = $cfg->getString('tryFirstAuthPass', 'simpleSAMLphp');
+        $this->serverconfig['SSO'] = $cfg->getBoolean('SSO', true);
```

Comment:
Wrong indentation, spaces instead of tabs.

Reply:
changed.